packages/graphql/src/lib/util.ts

```diff
@@ -8,4 +11,35 @@ export function shuffle<T>(array: Array<T>): Array<T> {
   return a
 }
 
+
+export function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
+export function checkCronToken(ctx: Context) {
+  if (typeof process.env.CRON_TOKEN === 'undefined') {
+    throw new GraphQLError("No token available.")
+  }
+  
+  if (ctx.req.headers["x-token"] !== process.env.CRON_TOKEN) {
+    throw new GraphQLError("Token not valid.")
+  }
+}
+
+export function formatDate(dateTime: Date) {
```

Comment:
should not work with js dates (too much of a pain). instead, use dayjs to format the date which is already installed